unity-client/Assets/Rendering/Shaders/Toon/Compiled/DCLSimpleLighting.hlsl

```diff
@@ -37,16 +37,15 @@ half4 DCL_SimpleFragmentPBR(InputData inputData, SurfaceData surfaceData)
         surfaceData.occlusion = min(surfaceData.occlusion, aoFactor.indirectAmbientOcclusion * DCL_CUSTOM_AO_TOON_FACTOR);
         surfaceData.occlusion = max(surfaceData.occlusion, aoFactor.directAmbientOcclusion);
     #endif
-    half3 color = GlobalIllumination(brdfData, brdfDataClearCoat, surfaceData.clearCoatMask,
-                                     inputData.bakedGI, surfaceData.occlusion,
-                                     inputData.normalWS, inputData.viewDirectionWS);
+    
+    half3 color = surfaceData.albedo * surfaceData.occlusion;
 
 #ifdef _ADDITIONAL_LIGHTS_VERTEX
     color += inputData.vertexLighting * brdfData.diffuse;
 #endif
-
-
-    color += surfaceData.emission;
+    
+    const float DCL_EMISSION_MULTIPLIER = 4;
```

Comment:
Is anyway we can set this constant outside the file without taking much time? It is used in several place and I am worried that if we change it in 1 shader, we forget to do it in all the others